org.eclipse.oneofour.client.data.testing/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>org.eclipse.oneofour-parent</artifactId>
+        <groupId>org.eclipse.oneofour</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <name>/org.eclipse.oneofour.client.data.testing</name>
```

Comment:
The leading slash looks strange to me. Is that intentional?

Reply:
The slash and the rather unconventional naming is a remnant from the NeoSCADA project.

Reply:
It is? Please get rid of it :grin: Unless you want to keep it.